### PR TITLE
React to hosting changes

### PR DESCRIPTION
**Pull request by phenning**
Addresses #361 

@davidfowl @rustd @JunTaoLuo 

Fowler, you said there are more changes coming, but I'd like to get this in to get the repo off the floor to unblock other high priority items

I will address any additional feedback post merge.

### CODE REVIEW

File: src/Rules/StarterWeb/AI/IndividualAuth/project.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
 
   "commands": {
-    "web": "Microsoft.AspNet.Server.Kestrel",
+    "web": "$safeprojectname$",
```
Comment (JunTaoLuo) [inline src/Rules/StarterWeb/AI/IndividualAuth/project.json:32]: The reason why we removed multiple commands in our samples is to prepare for the move towards dotnet cli where commands don't exist. Maybe we should do the same here and remove `ef`.

File: src/Rules/StarterWeb/IndividualAuth/project.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   },
 
   "commands": {
-    "web": "Microsoft.AspNet.Server.Kestrel",
+    "web": "$safeprojectname$",
```
Comment (JunTaoLuo) [inline src/Rules/StarterWeb/IndividualAuth/project.json:31]: and here

File: src/Rules/StarterWeb/AI/IndividualAuth/project.json (hunk 1 above)
Comment (phenning) [inline src/Rules/StarterWeb/AI/IndividualAuth/project.json:32]: I'll leave this here for now, but I opened #364 to track

File: src/Rules/StarterWeb/IndividualAuth/project.json (hunk 2 above)
Comment (phenning) [inline src/Rules/StarterWeb/IndividualAuth/project.json:31]: #364 tracking

**Comment by JunTaoLuo** [comment]
I didn't see `hosting.json` added for templates under `src/Rules` is that expected?

**Comment by phenning** [comment]
@JunTaoLuo By design, since it doesn't change from the base template version for any rule.

**Comment by JunTaoLuo** [comment]
@phenning other than possibly removing the `ef` commands, the rest is good.

**Comment by mikes-gh** [comment]
Isn't the web/run/kestrel command going away too for rc2?
Isn't it all dealt with by the new hosting model now?

**Pull request merged**